src/routes/github/github-router.test.ts

```diff
@@ -194,19 +194,6 @@ describe("GitHub router", () => {
 					expect.anything()
 				);
 			});
-			it("should not match route, return empty if uuid present but invalid", async ()=>{
```

Comment:
Turns out this test doesn't make sense. If there's random-gibberish in the path, `GitHubServerAppMiddleware` will treat it as cloud coz uuid is undefined.